<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-06-01T07:16:19 IST" name="Suite" finished-at="2022-06-01T07:17:13 IST" duration-ms="53602">
    <groups>
    </groups>
    <test started-at="2022-06-01T07:16:19 IST" name="QaacedemyTest" finished-at="2022-06-01T07:17:13 IST" duration-ms="53602">
      <class name="examples.endtoendframework.TestMain">
        <test-method signature="loginPageTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:examples.endtoendframework.TestMain@71ae31b0]" started-at="2022-06-01T07:16:19 IST" name="loginPageTest" data-provider="getDataProvider" finished-at="2022-06-01T07:16:36 IST" duration-ms="16736" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[manju]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[manju@123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Not Restricted]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPageTest -->
        <test-method signature="loginPageTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:examples.endtoendframework.TestMain@71ae31b0]" started-at="2022-06-01T07:16:36 IST" name="loginPageTest" data-provider="getDataProvider" finished-at="2022-06-01T07:17:11 IST" duration-ms="35732" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[manjunath]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[manju@123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Restricted]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at examples.endtoendframework.TestMain.loginPageTest(TestMain.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:425)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:68)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPageTest -->
        <test-method is-config="true" signature="closing()[pri:0, instance:examples.endtoendframework.TestMain@71ae31b0]" started-at="2022-06-01T07:17:12 IST" name="closing" finished-at="2022-06-01T07:17:13 IST" duration-ms="579" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closing -->
      </class> <!-- examples.endtoendframework.TestMain -->
    </test> <!-- QaacedemyTest -->
  </suite> <!-- Suite -->
</testng-results>
